src/node/Launcher.ts

```diff
@@ -146,15 +146,9 @@ class ChromeLauncher implements ProductLauncher {
 
       chromeExecutable = executablePathForChannel(channel);
     } else if (!executablePath) {
-      // Use Intel x86 builds on Apple M1 until native macOS arm64
```

Comment:
I might have missed the point, but this comment seems outdated because `/usr/bin/chromium-browser` isn't always an x86 build and the comment does not match the condition `!== 'darwin'`

Reply:
It looks it was introduced in https://github.com/puppeteer/puppeteer/pull/7099